kornia/color/gray.py

```diff
@@ -50,7 +54,62 @@ def rgb_to_grayscale(input: torch.Tensor) -> torch.Tensor:
         raise ValueError("Input size must have a shape of (*, 3, H, W). Got {}"
                          .format(input.shape))
 
-    # https://docs.opencv.org/4.0.1/de/d25/imgproc_color_conversions.html
     r, g, b = torch.chunk(input, chunks=3, dim=-3)
     gray: torch.Tensor = 0.299 * r + 0.587 * g + 0.110 * b
     return gray
+
+
+class BgrToGrayscale(nn.Module):
+    r"""convert image to grayscale version of image.
```

Comment:
```suggestion
    r"""convert BGR image to grayscale version of image.
```

Similarly in the `RgbToGrayscale`, we could now specify `convert RGB image to grayscale version of image.`

Because otherwise both modules have the same description

Reply:
yeah that's right, okay I'll fix it